aten/src/ATen/templates/Functions.cpp

```diff
@@ -36,7 +37,7 @@ Tensor TensorMaker::make_tensor() {
      data_ptr = makeDataPtrFromContext();
    }
 
-   Storage storage{Storage::use_byte_size_t{}, size_bytes, std::move(data_ptr)};
+   Storage storage{Storage::use_byte_size_t{}, size_bytes, std::move(data_ptr), /*allocator=*/c10::GetAllocator(c10::kMeta), /*resizeable=*/resizeable_};
```

Comment:
A nullptr allocator is not the same as Meta allocator... 
I'm not sure what is the impact of always giving an allocator here tbh.
Should we only pass one if resizable_ is true to not change current behavior?

Reply:
> Should we only pass one if resizable_ is true to not change current behavior?

yep sounds good!